apps/src/aichat/views/presentation/PresentationView.tsx

```diff
@@ -1,40 +1,65 @@
 import React from 'react';
 import ModelCardRow from './ModelCardRow';
-import {MODEL_CARD_FIELDS_AND_LABELS} from '@cdo/apps/aichat/views/modelCustomization/constants';
+import {
+  ALL_MODEL_CARD_FIELDS_LABELS_ICONS,
+  TECHNICAL_INFO_FIELDS,
+} from '@cdo/apps/aichat/views/modelCustomization/constants';
 import styles from '@cdo/apps/aichat/views/model-customization-workspace.module.scss';
 import {Heading4} from '@cdo/apps/componentLibrary/typography';
 import moduleStyles from './presentation-view.module.scss';
+import {useAppSelector} from '@cdo/apps/util/reduxHooks';
 
 const PresentationView: React.FunctionComponent = () => {
+  const currentAiCustomizations = useAppSelector(
+    state => state.aichat.currentAiCustomizations
+  );
+  const modelCardInfo = currentAiCustomizations.modelCardInfo;
+
+  // These are temporary constants. They will be retrieved from s3.
+  const EXAMPLE_MODEL_NAME = 'Model A';
+  const EXAMPLE_MODEL_TRAINING_DATA = 'Model A Training Data';
+
+  const TECHNICAL_INFO_VALUES: (string | number | boolean)[] = [
+    EXAMPLE_MODEL_NAME,
+    EXAMPLE_MODEL_TRAINING_DATA,
+    currentAiCustomizations.systemPrompt,
+    currentAiCustomizations.temperature,
+    currentAiCustomizations.retrievalContexts.length > 0,
+  ];
+
+  const getTechnicalInfo = () => {
```

Comment:
Seems like a good candidate for a `useMemo` - that way it should only update when `currentAiCustomizations` changes